### ISSUE TITLE
Change bundler from rollup to vite

### ISSUE THREAD

**Issue by NicolasOmar** [NicolasOmar/reactive-bulma#489]
### Objective

Change the main bundler tool from rollup implementation to vite.

---

### Description

After several failed attempts to update rollup plugin dependencies and checking that there are no solutions nowadays, I decided to review actual alternatives that will avoid this kind of situation in future releases and maintenance rounds.
The option I will review in this ticket is vite.

---

### Tasks

- [x] Study Vite's bundler mode implementation.
- [x] Remove rollup's implementation.
- [x] Change build script to execute vite's.
- [x] Check that the new `build` script is working.
- [x] Check that the new `lint` script is working.
- [x] Check that the new `test` script is working.
- [x] Check that the new `storybook build` script is working.
- [ ] Send this renewed method to the next minor version (v4.1.0).

**Comment by NicolasOmar** [comment]
New findings to be added as gattered knowledge:
- A specific issue regarding vite implementation on the current project failed due to a `vite-plugin-dts` build error.
  - I found the solution on a [vite-plugin-dts issue](https://github.com/qmhc/vite-plugin-dts/issues/388).
- Then, I had a problem with my storybook build script at the pipeline level (it worked locally).
  - The solution was to uninstall all storybook-related dependencies, run the `npx storybook@latest init` command on the project, and adjust its configurations to mine.
- Last but not least, my linter configuration started to break after I initiated this ticket. Appears to be a [common issue that is being solved](https://github.com/typescript-eslint/typescript-eslint/issues/10338#issuecomment-2481198019) as I am writing this explanation. I hope this could be solved during this week
  - To continue working, I commented on the linter step on my quality pipeline until a new version or configuration solves the issue.
